models/auth/oauth2_test.go

```diff
@@ -13,7 +13,12 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-//////////////////// Application
+func TestApplicationTableName(t *testing.T) {
+	e := unittest.GetXORMEngine()
+	tableInfo, err := e.TableInfo(new(auth_model.OAuth2Application))
+	assert.NoError(t, err)
+	assert.EqualValues(t, "oauth2_application", tableInfo.Name)
+}
```

Comment:
Don't see it really makes sense. It is just testing the XORM's mechanism.

Reply:
99fef44